added a section about fine tuning pupil detection settings (#245)
pupil-labs/pupil-docs: getting-started/capture-workflow.md

	4. Don't - The eye is out of focus.
</aside>

#### Settings Finetuning

> As a first step it is recommended to set the resolution, as some parameters are resolution dependent. 

Reviewer: ```suggestion
As a first step it is recommended to check the eye camera resolution, as some parameters are resolution dependent. 
```

> For stable and fast tracking we recommend 192x192 (or 320x240 depending on your hardware). 

Reviewer: ```suggestion
For robust and fast pupil detection and tracking we recommend using the default resolution settings. For 200hz eye cameras the default resolution is set to 192x192 pixels. If you have an older 120hz eye camera, the default is be 320x240 pixels.
```

> Pupil Capture offers the `Algorithm Mode` under `General Settings`. 

Reviewer: ```suggestion
In Pupil Capture you can view a visualization of the pupil detection Algorithm in the eye windows. For fine-tuning switch to this mode: `General Settings > Algorithm Mode`. 
```

> The algorithm display mode overlays a visualization of the pupil detection parameters on top of the eye video, which helps a lot during fine tuning most parameters.

Reviewer: ```suggestion

```

> **UVC Source / Sensor Settings**

Reviewer: ```suggestion
**Sensor Settings**
```

> * `Resolution`: 192x192 or 320x240 (depending on hardware)

Reviewer: ```suggestion
* `Resolution`: 192x192 for 200hz eye cameras. 320x240 for 120hz eye cameras.
```

* `Absolute Exposure Time`: Make sure that there is a high contrast between the pupil and its surrounding. In order to do so, you might need to increase brightness of the images. Start by testing one of these `Absolute Exposure Time` values: 64, 94, 124.

**Pupil Detector 2D/3D**

> * `Pupil Min/Max` : Change to `Algorithm Mode`. The bottom left circles (2 red, 1 green) represent the min, max and the current pupil sizes. Set the min and max values such that they include the current pupil size at all times in *all* poses.

Reviewer: ```suggestion
* `Pupil Min/Max` : Change to `General > Algorithm Mode`. The two red circles represent the min and max pupil size settings. The green circle visualizes the current apparent pupil size. Set the min and max values so the green circle (current pupil size) is within the min/max range for _all_ eye movements.
```

> Keep in mind that these values are defined in pixels and are therefore resolution dependent.

Reviewer: ```suggestion
<aside class="notice">
Keep in mind that pupil size values are defined in pixels and are therefore resolution dependent.
</aside>
```

* `Intensity Range` : Defines the minimum "darkness" of a pixel to be considered as the pupil.

> In `Algorithm Mode`, the considered pixels are visualized in blue. Try to minimize the range in a way that the pupil is always fully covered while having as little leakage as possible outside of the pupil.

Reviewer: ```suggestion
The pixels considered for pupil detection are visualized in blue within the `Algorithm Mode`. Try to minimize the range so that the pupil is always fully covered while having as little leakage as possible outside of the pupil.
```
